### PR TITLE
Support url parameters in track calls

### PR DESCRIPTION
**Pull request by bbeaird**
We're not sure why this other test started failing. It seems totally unrelated to our changes. Thoughts?

### CODE REVIEW

File: test/universal.js [hunk 1]
```diff
@@ -21,87 +21,91 @@ describe('Google Analytics :: Universal', function(){
...
         test.maps('completed-order-basic', settings);
       });
 
-      it('should map context.app', function(){
+      it('should map context.app', function() {
         test.maps('completed-order-app', settings);
       });
 
-      it('should map context.screen', function(){
+      it('should map context.screen', function() {
```
Comment (f2prateek) [inline test/universal.js]: What are these style changes for?

Comment (ndhoule) [inline test/universal.js]: fits the style enforced in other projects, we just haven't added linting to SS ints yet

**Comment by ndhoule** [comment]
@bbeaird If you remove these changes does the test still fail?

File: test/universal.js (hunk 1 above)
Comment (bbeaird) [inline test/universal.js]: The style changes were an accident. I thought the linter preferred the space after `()` though, yea?

Comment (ndhoule) [inline test/universal.js]: it does prefer the space. making only relevant changes in a PR helps make it be more readable, but totally nbd, these are fine changes to make :)

**Comment by f2prateek** [comment]
master builds fine so I would think it's from something in the PR https://circleci.com/gh/segmentio/integration-google-analytics/tree/master

**Pull request closed without merging**